index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+if (process.env.NEW_RELIC_ENABLED !== 'false') {
+  process.env.NEW_RELIC_HOME = __dirname;
+  require('newrelic');
+}
```

Comment:
I would expose `module.exports = null;` and `module.exports = require('newrelic')` depending on the cases. This will allow stuff like: https://github.com/transcovo/stargate/blob/master/src/app.js#L76